opencti-platform/opencti-front/src/components/InputSliderField.tsx

```diff
@@ -0,0 +1,127 @@
+import { Field, FieldProps } from 'formik';
+import React, { FunctionComponent } from 'react';
+import { Slider } from '@mui/material';
+import TextField from './TextField';
+import { SubscriptionFocus } from './Subscription';
+import { useLevel } from '../utils/hooks/useScale';
+import { Option } from '../private/components/common/form/ReferenceField';
+import { useFormatter } from './i18n';
+
+interface InputSliderFieldProps {
+  label: string;
+  variant?: string;
+  onSubmit?: (name: string, value: string | number | number[] | Option) => void;
```

Comment:
Don't think you can take option here. Why string ?

Reply:
We removed the Option, but we need the string because the Slider component send a number and for the mutation to append, we need to pass string